tests/fixture/subnet/xsvm.go

```diff
@@ -33,6 +33,10 @@ func NewXSVMOrPanic(name string, key *secp256k1.PrivateKey, nodes ...*tmpnet.Nod
 
 	return &tmpnet.Subnet{
 		Name: name,
+		Config: tmpnet.FlagsMap{
+			// Reducing this from the 1s default speeds up tx acceptance
+			"proposerMinBlockDelay": 0,
+		},
```

Comment:
Is this a no-op?

Reply:
Why include a non no-op config change in this PR? Just looking to understand here